app/models/game.rb

```diff
@@ -1,8 +1,19 @@
 class Game < ActiveRecord::Base
-
+  belongs_to :user
+  COMPLEXITIES = {
+    1 => "low",
+    2 => "medium",
+    3 => "high",
+    4 => "extreme"
+  }
   validates :name, presence: true, uniqueness: true
+  validates :user_id, presence: true
   validates :description, presence: true
   validates :min_players, presence: true
+  validates :min_players, numericality: {
+    less_than_or_equal_to: :max_players, message:
+    "must be less than or equal to max players!"
+    }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.